### DISCUSSION TITLE
What about HH41V?

### DISCUSSION THREAD

**Discussion by DjordjeMandic** [froonix/HH40V#7] [General]
I've got HH41V, is there any easy way to support it in openwrt?

**Comment by killerbees19** [comment]
Crosspost: https://forum.openwrt.org/t/support-for-alcatel-hh40v/106028/14

**Comment by DjordjeMandic** [comment]
I have opened new topic just for HH41V, to me they look the same

https://forum.openwrt.org/t/support-for-alcatel-hh41v/186376

**Reply by DjordjeMandic** [reply]
Stable support? Probably not by me. I don't know how to port openwrt properly. Currently I just used image for 40 instead of 41. Read forum posts on link above.

I will be putting it to the test for basic wifi + lan in month or two. Modem does not work

**Reply by DjordjeMandic** [reply]
> thank you.
> and i hope that you can test the latest release of openwrt for 40 with 41.

I will try to test basic AP functionality but can't guarantee exactly when. 

> for the modem: did you try to configure it like described here: LTE-Modem configuration
> https://openwrt.org/toh/alcatel/hh40v
> ?

No, i did not. I don't have card for it yet. But it would be useful for me so there is good probability that i will try to configure it

**Reply by DjordjeMandic** [reply]
I’ve tested the latest OpenWRT release, Wi-Fi is working, and the ART partition is a 1:1 match (identical binary). LAN ports are functional as well. I haven’t tested the modem yet since I currently don’t have a SIM card for that device, but I might give that a try next month with prepaid card.

One thing to note: some of the LAN port LEDs aren’t working correctly, which I suspect is due to JTAG-related issues. Otherwise, basic LAN and Wi-Fi functionality are solid.
